### PR TITLE
Add container workload information; Split sections out into separate files

### PR DESCRIPTION
**Pull request by scottmalkie**
*Issue #, if available:* N/A

*Description of changes:*
Split the various sections out into separate files, added a first pass of information/links on container-based workloads.
Feedback welcome! Thanks!

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: containers.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+If you are currently running only x86_64-based container images, the first step is leveraging Graviton is to switch to [multiple-architecture (multi-arch)](https://www.docker.com/blog/multi-platform-docker-builds/) images. Most container registries including [Docker Hub](https://hub.docker.com) and [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/blogs/containers/introducing-multi-architecture-container-images-for-amazon-ecr/) now support multi-arch images.
+
+There are a couple methods of creating a multi-arch image.
+
+1. [Docker Buildx](https://github.com/docker/buildx#getting-started)
+2. Using a CI/CD Build Pipeline such as [Amazon CodePipeline](https://github.com/aws-samples/aws-multiarch-container-build-pipeline) to coordinate native build and manifest generation.
+
+## Deploying to Graviton
+
+Most container orchestration platforms support both aarch64 and x86_64 hosts.
```
Comment (AGSaidi) [inline containers.md:15]: we used arm64 throughput the doc.

File: java.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+# Java on Graviton
+
```
Comment (AGSaidi) [inline java.md:2]: need an intro paragraph about how most java is cross platform and runs well.

Maybe also something about corretto

File: containers.md (hunk 1 above)
Comment (scottmalkie) [reply containers.md]: changed to arm64

File: java.md (hunk 2 above)
Comment (scottmalkie) [reply java.md:2]: added in intro paragraphs to the java and python pages, added a blurb about corretto as well

**Pull request merged**